mk/spksrc.service.installer.functions

```diff
@@ -48,11 +47,8 @@ initialize_variables ()
 
     # Extract share volume and share name from download location if provided
     if [ -n "${SHARE_PATH}" ]; then
-        if [ -n "${wizard_volume}" ]; then
-            SHARE_PATH="${wizard_volume}/${SHARE_PATH}"
-        fi
-        SHARE_VOLUME=$(echo "${SHARE_PATH}" | awk -F/ '{print "/"$2}')
-        SHARE_NAME=$(echo "${SHARE_PATH}" | awk -F/ '{print $3}')
+        SHARE_VOLUME=$(echo "$(realpath ${SHARE_PATH})" | awk -F/ '{print "/"$2}')
+        SHARE_NAME=$(echo $(realpath ${SHARE_PATH}) | awk -F/ '{print $3}')
```

Comment:
I find abspath often more appropriate as realpath returns nothing if the path doesn't exists.  As such this can pose problems for path to be created shortly there after.

Reply:
But on DSM, `abspath` is not available, only `realpath` is.